rpa-builder/6.7/modules/ROOT/pages/toolbox-mail-operations-mail-session-oauth.adoc

```diff
@@ -0,0 +1,139 @@
+= Mail Session (OAuth)
+
+The _Mail Session (OAuth)_ Action Step is an inclusive element for email operations that enables you to establish a connection to an email server using MS Outlook or Gmail and then perform email operations such as *Read Mail*, *Send Mail*, and *Set Mail* (to change email) by using the corresponding Action Steps.
+
+== Before you Begin
+
+You must have an Azure account to create an application for Outlook.
```

Comment:
Enhance this for both MS Outlook and Gmail, for both users must create an application